src/misc/setup.cpp

```diff
@@ -983,18 +984,26 @@ const Section_line &Config::GetOverwrittenAutoexecSection() const
 
 bool Config::ParseConfigFile(const std::string &type, const std::string &configfilename)
 {
+	std::error_code ec;
+	std_fs::path cfg_path = configfilename;
+	std_fs::path conanical_path = std_fs::canonical(cfg_path, ec);
+	if (ec)
+		return false;
+	
+	if (set_contains(loadedConfigFIles, conanical_path.string())) {
+		LOG_INFO("CONFIG: Skipping duplicate config file '%s'", 
+			configfilename.c_str());
+		return true;
+	}
 	// static bool first_configfile = true;
-	ifstream in(configfilename);
-	if (!in)
+	ifstream in(conanical_path);
+	if (!in) 
 		return false;
```

Comment:
With `conanical_path` being a fully-constructed `path` object, we can access it's `stat()` bit for free using: `if (!std_fs::exists(conanical_path))`, instead of co-opting the stream open call to tell the same info.  (https://stackoverflow.com/a/12774387).

Reply:
Thanks; I didn't realize we used `in` further down for actual reading.  (resolving!)